frontend/src/polyfills.ts

```
@@ -0,0 +1,9 @@
+if (typeof document.startViewTransition !== 'function') {
+  document.startViewTransition = (callback: () => Promise<void>): ViewTransition => ({
+    ready: callback(),
+    skipTransition: null!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)